localstack/services/sqs/models.py

```diff
@@ -881,6 +873,23 @@ def _on_remove_message(self, message: SqsMessage):
                 # this may happen if the message no longer exists because it was removed earlier
                 pass
 
+    def validate_queue_attributes(self, attributes):
+        valid = [
+            k[1]
+            for k in inspect.getmembers(
+                QueueAttributeName, lambda x: isinstance(x, str) and not x.startswith("__")
```

Comment:
purely curious, what's the reasoning behind the startswith clause, which attributes are weeded out here?

Reply:
`get_members` returns all of the members and dunder methods, after filtering for `str`, I believe it is `__name__` that was left.